crates/fuel-core/src/database/balances.rs

```diff
@@ -61,24 +56,8 @@ impl StorageMutate<ContractsAssets> for Database {
 }
 
 impl MerkleRootStorage<ContractId, ContractsAssets> for Database {
-    fn root(&self, parent: &ContractId) -> Result<MerkleRoot, Self::Error> {
-        let items: Vec<(Vec<u8>, Word)> = Database::iter_all_by_prefix(
-            self,
-            Column::ContractsAssets,
-            Some(parent),
-            Some(IterDirection::Forward),
-        )
-        .try_collect()?;
-
-        let root = items
-            .iter()
-            .filter_map(|(key, value)| {
-                (&key[..parent.len()] == parent.as_ref()).then_some((key, value))
-            })
-            .sorted_by_key(|t| t.0)
-            .map(|(_, value)| value.to_be_bytes());
-
-        Ok(crypto::ephemeral_merkle_root(root).into())
+    fn root(&self, _: &ContractId) -> Result<MerkleRoot, Self::Error> {
+        Ok(*empty_sum_sha256())
```

Comment:
Why are we doing this again?

Reply:
To be sure that we don't calculate root in any case for the beta 3=)